ketcher-autotests/tests/utils/formats/formats.ts

```diff
@@ -103,9 +103,10 @@ export async function getSdf(
 export async function setMolecule(
   page: Page,
   structStr: string,
+  rescale: boolean,
```

Comment:
Perhaps as `rescale` parameter defaults to `true` in `setMolecule` API it makes sense to set it so in this helper as well